graphqlapi/src/main/java/io/stargate/graphql/web/HttpAwareContext.java

```diff
@@ -48,70 +36,14 @@
   // For more information.
   private final BatchContext batchContext = new BatchContext();
 
-  private static final String HEADER = "Authorization";
-  private static final String PREFIX = "Bearer ";
-
-  public HTTPAwareContextImpl(
-      DataLoaderRegistry dataLoaderRegistry,
-      HttpServletRequest request,
-      HttpServletResponse response) {
-    this.dataLoaderRegistry = dataLoaderRegistry;
+  public HttpAwareContext(HttpServletRequest request) {
     this.request = request;
-    this.response = response;
-  }
-
-  // Web socket
-  public HTTPAwareContextImpl(
-      DataLoaderRegistry dataLoaderRegistry, Session session, HandshakeRequest handshakeRequest) {
```

Comment:
I'm not sure if/how web socket worked, it was definitely not covered by the tests.

It's the only thing we lose from moving away from graphql-servlet, I can look at re-adding it if it's something we want to pursue.